### ISSUE TITLE
Confusing/broken sufficient techniques in 2.4.11 and 2.4.12

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#3455]
In 2.4.11 and 2.4.12 the Sufficient technique is shown as:

<ol>
  <li> <a href="https://www.w3.org/WAI/WCAG22/Techniques/css/C43">C43: Using CSS scroll-padding to un-obscure content</a> Using CSS margin and scroll-margin to un-obscure content  </li>
</ol>

I assume the “Using CSS margin and scroll-margin to un-obscure content” text comes from the text outside the link in the source code [here (2.4.11)](https://github.com/w3c/wcag/blob/main/understanding/22/focus-not-obscured-minimum.html#L184) and [here (2.4.12)](https://github.com/w3c/wcag/blob/main/understanding/22/focus-not-obscured-enhanced.html#L67).

While fixing this in #3454, I wondered where the text inside of the link comes from, and it comes indeed from this technique in the source code: https://github.com/w3c/wcag/blob/4dfd63d49c148007570403bc108207b8662b8b22/techniques/css/C43.html

I assume that the technique was using margin and scroll margin and then was revised to use scroll-padding. The old file however seems to be sticking around, hence you get a weird situation where a link mentioning scroll-padding goes to a page explaining scroll-margin.

- [ ] Merge PR #3454 (when done).
- [ ] Make sure to publish the correct technique pages.

**Comment by fstrr** [comment]
You're right that the original technique used the margin properties. After some feedback, it was updated to use the padding properties. There seems to be a fair amount of outdated files sticking around for some reason. I mentioned this particular one to @alastc in last Friday's backlog meeting.

Thanks for working on it.

**Comment by fstrr** [comment]
Going back to look at this, I'm seeing different content to you.

If I go to the [Focus Not Obscured (Min) Understanding document](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html), I see the link text as using "padding" rather than "margin":

<img width="1044" alt="screen grab of the link to the C43 technique showing the link text using scroll-padding" src="https://github.com/w3c/wcag/assets/20049493/347bcbdf-1e40-49d3-8c0f-3dd267a7112c">

When I click on the link to see the [C43 technique](https://www.w3.org/WAI/WCAG22/Techniques/css/C43), I see the old "margin" technique:

<img width="1102" alt="screen grab of page title of the C43 technique that uses scroll-margin" src="https://github.com/w3c/wcag/assets/20049493/f9cd57c2-bdf5-47a7-a840-e26457edbb88">

The same thing happens with the "enhanced" criterion.

**Comment by yatil** [comment]
@fstrr You see the same thing that I saw. 😃 It's very confusing.

**Comment by kfranqueiro** [comment]
This was republished as of mid-December 2023 at the latest (judging by web.archive.org).
